src/sass/_theme_rst.sass

```diff
@@ -200,14 +200,14 @@
   // This is the #href that shows up on hover. Sphinx's is terrible so I hack it away.
   h1, h2, h3, h4, h5, h6, dl dt, p.caption, table > caption, .code-block-caption
     .headerlink
-      visibility: hidden
-      font-size: 14px
+      opacity: 0
+      font-size: 20px
+      margin-left: 0.5em
       @extend .fa
-      &:after
-        content: "\f0c1"
-        font-family: FontAwesome
```

Comment:
Currently, the rendering does not match how it did previously because the font family should still be FontAwesome.

Also is there a reason behind the font size change and the adjusted margin?

Reply:
After losing the FontAwesome font family, I was adjusting the size and margin to match the original size better.

However, as you suggested, I put FontAwesome back to match the prior style even better. Did have to include a little left margin because the link icon was pressed right against the text otherwise when not in :after content.